### PR TITLE
Add org.metamuffin.hurrycurry.client

### PR DESCRIPTION
**Pull request by metamuffin**
### Please confirm your submission meets all the criteria

- [x] Please describe the application briefly.
  A cooperative fast-paced game about running a restaurant together.

- [x] The domain used for the application ID is [controlled by the application developers][appid-domain] and the [application id guidelines][appid] are followed.
- [x] I have read the and followed all the [Submission and licence requirements][reqs].
- [x] I have [built][build] and tested the submission locally.
- [x] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about this submission. **Link:**

[appid-domain]: https://docs.flathub.org/docs/for-app-authors/requirements#control-over-domain
[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

File: org.metamuffin.hurrycurry.client.desktop
Comment (bbhtt) [inline org.metamuffin.hurrycurry.client.desktop:1]: This should be upstream

File: org.metamuffin.hurrycurry.client.metainfo.xml [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+            <image>https://img.itch.zone/aW1hZ2UvMjgxOTEyNy8xNjg2MzQzMC5qcGVn/original/YDCBug.jpeg</image>
+        </screenshot>
+        <screenshot>
+            <image>https://img.itch.zone/aW1hZ2UvMjgxOTEyNy8xNjg2MzQzMy5qcGVn/original/ynbKBP.jpeg</image>
+        </screenshot>
+        <screenshot>
+            <image>https://img.itch.zone/aW1hZ2UvMjgxOTEyNy8xNjg2MzQzMi5qcGVn/original/CT6oZD.jpeg</image>
+        </screenshot>
+        <screenshot>
+            <video codec="av1" container="webm">
```
Comment (bbhtt) [inline org.metamuffin.hurrycurry.client.metainfo.xml:74]: videos won't work.

File: org.metamuffin.hurrycurry.client.metainfo.xml
Comment (bbhtt) [inline org.metamuffin.hurrycurry.client.metainfo.xml:1]: This should be upstream

**Comment by metamuffin** [comment]
We applied the proposed changes.

**Comment by bbhtt** [comment]
bot, build org.metamuffin.hurrycurry.client

File: org.metamuffin.hurrycurry.client.yaml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        url: https://downloads.tuxfamily.org/godotengine/4.2.2/godot-4.2.2-stable.tar.xz
+
+      - type: file
+        path: custom.py
+
+    build-commands:
+      - python3 /app/bin/scons $SCONS_FLAGS_EXTRA profile=custom.py target=editor -j "$FLATPAK_BUILDER_N_JOBS"
+      - python3 /app/bin/scons $SCONS_FLAGS_EXTRA profile=custom.py target=template_release -j "$FLATPAK_BUILDER_N_JOBS"
+      - install -Dm755 bin/godot.linuxbsd.template_release.* /app/bin/godot
+      - install -Dm755 bin/godot.linuxbsd.editor.* /app/bin/godot-editor
```
Comment (hfiguiere) [inline org.metamuffin.hurrycurry.client.yaml:54]: do you need to install the editor?

Comment (hfiguiere) [reply org.metamuffin.hurrycurry.client.yaml:54]: nevermind.

**Pull request closed without merging**